app/src/main/res/values/strings.xml

```diff
@@ -83,4 +86,6 @@
     <string name="prefix_audio_file_name">RecordingAudio_</string>
     <string name="prefix_uploading">Uploading: </string>
     <string name="msg_get_user_info_failed"> .Please try to login again!</string>
+    <string name="description_back_to_timeline">Press to back to timeline</string>
+    <string name="description_vote_for_post">Vote for this post</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.description_vote_for_post` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
This will be of use in my next pull request.